actions/requirements-actions.txt

```diff
@@ -5,4 +5,5 @@ beautifulsoup4~=4.6.3
 requests~=2.21.0
 geopy~=1.18.1
 pandas~=0.24.1
-algoliasearch>=2.0,<3.0
\ No newline at end of file
+algoliasearch>=2.0,<3.0
+spacy~=2.1.9
```

Comment:
bahh i wish we didn't need to have spacy as a dependency it's so large 🙈 ah well

Reply:
We could use a different stopword list if we wanted to get rid of it?